### PR TITLE
Make lookup plugins return lists to fix failures

### PR DESCRIPTION
**Pull request by AlanCoding**
##### SUMMARY
Trying to fix CI failures.

##### ISSUE TYPE
 - Bug, Docs Fix or other nominal change

##### COMPONENT NAME
 - API

### CODE REVIEW

File: awx_collection/README.md [hunk 1]
```diff
@@ -32,33 +32,11 @@ Installing the `tar.gz` involves no special instructions.
...
- - a config file at `/etc/tower/tower_cli.cfg`
-
-Config file syntax looks like this:
-
-```
-[general]
-host = https://localhost:8043
-verify_ssl = true
-oauth_token = LEdCpKVKc4znzffcpQL5vLG8oyeku6
-```
```
Comment (AlanCoding) [inline awx_collection/README.md:61]: @tvo318 You made this change to `awx_collection/tools/roles/template_galaxy/templates/README.md.j2` but this file is templated from that file.

We have this duplication due to technical reasons, allowing installing from source. This is obtained from running `make awx_collection_build`, so I want to be super clear that this unrelated change comes from the templating.

File: awx_collection/README.md [hunk 2]
```diff
@@ -101,6 +79,7 @@ The following notes are changes that may require changes to playbooks:
  - The `notification_configuration` parameter of `tower_notification_template` has changed from a string to a dict. Please use the `lookup` plugin to read an existing file into a dict.
  - `tower_credential` no longer supports passing a file name to `ssh_key_data`.
  - The HipChat `notification_type` has been removed and can no longer be created using the `tower_notification_template` module.
+ - Lookup plugins now always reutrn a list, and if you want a scalar value use `lookup` as opposed to `query`
```
Comment (webknjaz) [inline awx_collection/README.md]:
```suggestion
 - Lookup plugins now always return a list, and if you want a scalar value use `lookup` as opposed to `query`
```

**Review by PabloHiro — APPROVED** [review]

File: awx_collection/README.md [hunk 3]
```diff
@@ -32,33 +32,11 @@ Installing the `tar.gz` involves no special instructions.
 ## Running
 
 Non-deprecated modules in this collection have no Python requirements, but
-may require the official [AWX CLI](https://pypi.org/project/awxkit/)
+may require the AWX CLI
```
Comment (thedoubl3j) [inline awx_collection/README.md:35]: If we are are referencing the CLI, in general, I think we would probably want to link to it?

**Review by chrismeyersfsu — APPROVED** [review]

File: awx_collection/README.md (hunk 3 above)
Comment (AlanCoding) [reply awx_collection/README.md:35]: Not my change, from templating.
